events/events-listen-timeouts.feature

```diff
@@ -24,6 +24,9 @@ Feature: Event Listen Timeouts
 		Then the client throws a "ACK_TIMEOUT" error with message "No ACK message received in time for eventPrefix/.*"
 
 	# The client unlistens to eventPrefix
+		When the client listens to events matching "eventPrefix/.*"
```

Comment:
Yeah I had just copied and pasted the connection steps from events.feature and events-listen.feature ...